py17track/package.py

```diff
@@ -278,3 +281,13 @@ def __attrs_post_init__(self):
         object.__setattr__(self, "origin_country", COUNTRY_MAP[self.origin_country])
         object.__setattr__(self, "package_type", PACKAGE_TYPE_MAP[self.package_type])
         object.__setattr__(self, "status", PACKAGE_STATUS_MAP[self.status])
+
+        if self.timestamp is not None:
+            timestamp_loc = timezone(self.tz).localize(
+                datetime.strptime(self.timestamp, "%Y-%m-%d %H:%M")
+            )
+            timestamp_utc = timestamp_loc.astimezone(timezone("UTC")).strftime(
+                "%Y-%m-%d %H:%M:%S"
+            )
+            object.__setattr__(self, "timestamp", timestamp_utc)
+            object.__setattr__(self, "tz", "UTC")
```

Comment:
This will always set this attribute to `"UTC"`, even if the user passes in a different value for `tz`. That said, I don't think we need this attribute, as one can get the timezone info via `self.timestamp.tzinfo`.

Reply:
No, timestamp is a string not a datetime object. self.timestamp.tzinfo is not available.

Reply:
I think we should store native types wherever possible; `timestamp` should be a `datetime` object so our downstream users can have more flexibility.

Reply:
Okay, agreed

Reply:
Now that `self.timestamp` is a `datetime` object, you don't need to store a separate, string-based representation of the timezone via `self.tz` – `self.timestamp.tzinfo` should suffice?